src/main/java/seedu/address/logic/Password.java

```diff
@@ -0,0 +1,66 @@
+package seedu.address.logic;
+
+import static java.util.Objects.requireNonNull;
+
+import seedu.address.commons.exceptions.IllegalValueException;
+
+//@@author jelneo
+/**
+ * Represents the password of a user account
+ */
+public class Password {
+    public static final int PASSWORD_LENGTH = 6;
```

Comment:
Perhaps you want to rename this? This constant name seems to suggest that the password length has to be exactly 6 characters long

Reply:
corrected